Userland/Games/Minesweeper/Field.h

```diff
@@ -45,20 +45,63 @@ class Field final : public GUI::Frame {
 public:
     virtual ~Field() override;
 
+    enum class Difficulty {
+        Beginner,
+        Intermediate,
+        Expert,
+        Madwoman
+    };
+
+    StringView difficulty_to_string(Difficulty difficulty)
+    {
+        switch (difficulty) {
+        case Difficulty::Beginner:
+            return "beginner";
+        case Difficulty::Intermediate:
+            return "intermediate";
+        case Difficulty::Expert:
+            return "expert";
+        case Difficulty::Madwoman:
+            return "madwoman";
+        }
+
+        VERIFY_NOT_REACHED();
+    }
+
+    Difficulty difficulty_from_string(StringView difficulty_string)
+    {
+        if (difficulty_string.matches("beginner"))
+            return Difficulty::Beginner;
+
+        if (difficulty_string.equals_ignoring_case("intermediate"))
+            return Difficulty::Intermediate;
+
+        if (difficulty_string.equals_ignoring_case("expert"))
+            return Difficulty::Expert;
+
+        if (difficulty_string.equals_ignoring_case("madwoman"))
+            return Difficulty::Madwoman;
+
+        VERIFY_NOT_REACHED();
```

Comment:
This crashes the program when putting any value that doesn't match the above in the config file. This should return `Optional<Difficulty>` instead and handle failure at the call site.